### ISSUE TITLE
Datepicker locale

### ISSUE THREAD

**Issue by 5im-0n** [owntracks/frontend#131]
Is it possible to somehow configure the datepicker locale? Having the week start on monday is making me crazy :)

I saw that vue2-datepicker supports locales[1], but am I right that at the moment there is no way to configure them with `config.js`?

[1]https://github.com/mengxiong10/vue2-datepicker?tab=readme-ov-file#internationalization

**Comment by jpmens** [comment]
> Having the week start on sunday is making me crazy

Many of us Europeans are very relieved the week starts on Monday!

**Comment by linusg** [comment]
> am I right that at the moment there is no way to configure them with config.js?

Yes. I'll add an option once I get around to it, in the future we can add automatic detection with https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/Locale/getWeekInfo
